package.json

```diff
@@ -14,31 +14,34 @@
     "deploy-network": "npx cdk deploy OpenTunaNetworkStack --app 'npx ts-node bin/opentuna-network.ts'"
   },
   "devDependencies": {
-    "@aws-cdk/assert": "1.56.0",
+    "@aws-cdk/assert": "1.57.0",
     "@iarna/toml": "^2.2.5",
     "@types/jest": "^25.2.1",
     "@types/mustache": "^4.0.1",
     "@types/node": "10.17.5",
-    "aws-cdk": "1.56.0",
+    "aws-cdk": "1.57.0",
```

Comment:
Once updating package version in package.json, pls also submit the package-lock.json to make sure the dependencies  pinned in expected version.